cli-commands/ganache/ganache.js

```diff
@@ -1,31 +1,35 @@
 const ganache = require('ganache-cli');
 const setup = require('./setup.json');
+let port;
 
 const run = (inPort, logger) => {
-	let port = (inPort) ? inPort : setup.defaultPort;
+	port = (inPort) ? inPort : setup.defaultPort;
 	const server = ganache.server({
 		accounts: setup.accounts,
 		logger
 	});
-	
-	server.listen(port, function (err, blockchain) {
 
-		if (err) {
-			console.log(err);
-			return;
-		}
+	server.listen(port, listenCallback);
+};
 
-		const accountsLength = blockchain.options.accounts.length;
+const listenCallback = (err, blockchain) => {
```

Comment:
This should probably be named better. Reading this variable it's not understandable that this is related with ganache